tutor/env.py

```diff
@@ -551,3 +550,11 @@ def _delete_plugin_templates(plugin: str, root: str, _config: Config) -> None:
                 raise exceptions.TutorError(
                     f"Could not delete file {e.filename} from plugin {plugin} in folder {path}"
                 )
+
+
+@hooks.Filters.IS_FILE_RENDERED.add()
+def _do_not_render_binary_files(result: bool, path: str) -> bool:
+    """Return bianry file"""
```

Comment:
nit: the docstring does not make sense. Please update it.

Reply:
concern addressed